docs/INSTALLATION.md

```diff
@@ -1,12 +1,9 @@
-Install [Node.js](http://nodejs.org):
+There are a couple of ways to install [NodeJS](http://nodejs.org):
 
-* **Windows and OS X users**: can
-[download an Installer](https://nodejs.org/en/download/)
-* **Linux users**: can install it
-[via package manager](https://nodejs.org/en/download/package-manager/)
-
-[linstall]: https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager
+* via an Installer or Binaries: https://nodejs.org/en/download/ 
```

Comment:
I'm not sure plain links will look very good in the page. Do you think making them *text with link* would make them better?

Sorry for my English, what do you think about making them `[Installer or Binaries](https://nodejs.org/en/download/)`?. They would look like [Installer or Binaries](https://nodejs.org/en/download/) instead of https://nodejs.org/en/download/

Reply:
Yup, this would be better. Same goes for `package managers`.

Reply:
No problem! I'll fix them up this evening when I get home.